doc/api/crypto.md

```diff
@@ -1044,6 +1044,76 @@ encoding of `'utf8'` is enforced. If `data` is a [`Buffer`][], `TypedArray`, or
 
 This can be called many times with new data as it is streamed.
 
+## Class: KeyObject
+<!-- YAML
+added: REPLACEME
+-->
+
+Node.js uses an internal `KeyObject` class which should not be accessed
+directly. Instead, factory functions exist to create instances of this class
+in a secure manner, see [`crypto.createSecretKey`][],
+[`crypto.createPublicKey`][] and [`crypto.createPrivateKey`][]. A `KeyObject`
+can represent a symmetric or asymmetric key, and each kind of key exposes
+different functions.
+
+Most applications should consider using the new `KeyObject` API instead of
+passing keys as strings or `Buffer`s due to improved security features.
+
+### keyObject.getType()
+<!-- YAML
+added: REPLACEME
+-->
+* Returns: {string}
+
+Depending on the type of this `KeyObject`, this function either returns
+`'secret'` for symmetric keys, `'public'` for public (asymmetric) keys or
+`'private'` for private (asymmetric) keys.
+
+### keyObject.getSymmetricSize()
+<!-- YAML
+added: REPLACEME
+-->
+* Returns: {number}
+
+For symmetric keys, this function returns the size of the key in bytes. This
```

Comment:
It is true that most sizes are measured in bits, but most crypto APIs I have come across have used bytes since more or less all practically relevant applications use a multiple of 8 bits anyway, and it is much more convenient to allocate, copy, transfer and free a number of bytes. I think all of our APIs express sizes as bytes, e.g. key derivation.

Reply:
OK, agree on returning bytes. Don't understand why there is only a getSymmetricSize() though. Once we have getAsymmetricSize(), people are going to be scratching their head wondering why there are two functions.

Reply:
I'd argue that they cannot be compared, and that their meaning is entirely different. When would you use these interchangeably?